data/json/items/gun/5x50.json

```diff
@@ -49,7 +53,11 @@
     "aim_speed" : 5,
     "recoil": 50,
     "durability": 9,
-    "burst": 5,
+    "modes" : [
+       [ "DEFAULT", "semi-auto", 1 ],
+       [ "BURST", "2 rd.", 2 ],
```

Comment:
I was using the burst fire mode to simulate firing both barrels at once. This is just an ordinary burst fire setting.